java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java

```diff
@@ -193,61 +202,96 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
 
         if (this.missingTime != null) {
             generator.writeKey("missing_time");
-            generator.write(this.missingTime);
-
+            this.missingTime.serialize(generator, mapper);
         }
+
         generator.writeKey("missing_time_in_millis");
         generator.write(this.missingTimeInMillis);
 
         generator.writeKey("missing_total");
         generator.write(this.missingTotal);
 
         if (this.time != null) {
-            generator.writeKey("time");
-            generator.write(this.time);
-
+            generator.writeKey("getTime");
```

Comment:
The `getTime` does not look right. shouldn't be `time`?:

```suggestion
            generator.writeKey("time");
```

Reply:
The field is actually `getTime`: https://github.com/opensearch-project/OpenSearch/blob/2.x/server/src/main/java/org/opensearch/index/get/GetStats.java#L158

Looks like this bug was introduced nearly twelve years ago: https://github.com/opensearch-project/OpenSearch/commit/cc83c2f848be69a77f1275fe1ff5363dcdd4c955#diff-f7abcc8f5eb3ec75bb9be11b7fe88baa0bcedf62c3c481552a0e98ceb7125a27R124

Reply:
Thanks @Xtansia

Reply:
Do you mind creating an issue on OpenSearch side?

Reply:
Created https://github.com/opensearch-project/OpenSearch/issues/16894